.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```

Comment:
Just want to confirm that this is necessary and not some sort of addition specific to your local setup.

Reply:
💯  you're right. Didn't realize the folder wasn't gitignored. My bad. Removing these.